posthog/models/feature.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Feature(models.Model):
+    class StatusType(models.TextChoices):
+        PRE_ALPHA = "pre_alpha", "Pre-alpha"
+        ALPHA = "alpha", "Alpha"
+        BETA = "beta", "Beta"
+        GENERAL_AVAILABILITY = "general_availability", "GA"
```

Comment:
It looks like Early Access features also have a status enum; 
```
    class Stage(models.TextChoices):
        DRAFT = "draft", "draft"
        CONCEPT = "concept", "concept"
        ALPHA = "alpha", "alpha"
        BETA = "beta", "beta"
        GENERAL_AVAILABILITY = "general-availability", "general availability"
        ARCHIVED = "archived", "archived"
```

What's your thinking on how you're reconciling the status of "Features" with "Early access features"?

Reply:
This is a great Q and I think we may want to discuss it tomorrow. It goes back to my original inclination of wanting to build this into early access features. For right now, I think we may actually want to scrap the Feature-level `status` field and instead reach for a "primary early access feature` field, from which we can ascertain the overall feature status, opt-in users, etc that are seen in the mockup

Reply:
I'll bump this PR down to draft status and push my proposal so we can look over it together